packages/convict/README.md

```diff
@@ -338,48 +453,30 @@ If you want to allow comments in your JSON file, use [JSON5](https://www.npmjs.c
 convict.addParser({extension: 'json', parse: require('json5').parse});
 ```
 
+## API convict (global)
 
-## API
+Some function are only global, like : addParser, addFormat, addGetter...
 
-### var config = convict(schema)
+### convict.addParser(parser)
 
-`convict()` takes a schema object or a path to a schema JSON file and returns a
-convict configuration object.
-
-The configuration object has an API for getting and setting values, described
-below.
+ - **parser**: is an Object or an Array contains containing Object.
```

Comment:
`contains containing Object` - not sure what did you mean here

Reply:
- **parser**: is an Object or an Array containing Objects. maybe better